extension/llm/modeling/text_decoder/rope.py

```diff
@@ -0,0 +1,308 @@
+# @lint-ignore-every LICENSELINT
```

Comment:
can we use `git mv` to preserve the history better? I saw the import error message update, perhaps that's why you didn't do it?